appendices/VK_IMG_format_pvrtc.txt

```diff
@@ -0,0 +1,13 @@
+include::meta/VK_IMG_format_pvrtc.txt[]
+
+=== New Enum Constants
+
+  * Extending elink:VkResult:
```

Comment:
This should be elink:VkFormat

Reply:
indeed.

PS: this stub appendix relates to #512. Should I add "IP Status: unresolved" or such?

Reply:
Ehhh... not for me to say. Is this change critical to any of the script changes you've made? If not, might be better to take it out.

Reply:
Nah, just sounds worse **not** to include the IP status.

The stub is important for the script though. It assumes appendix for each extension exists. Would otherwise need to add exception mechanism.